src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+
+/**
+ * Represents an exception thrown by Duke.
+ */
+public class DukeException extends Exception {
+    public DukeException(String message) {
+        super(message);
+    }
+
+    /**
```

Comment:
I think the header comment here can be omitted since its a simple overriding method